FAILED ensemble_evaluator/prefect/test_prefect_ensemble.py::test_prefect_retries

**Describe the bug**
`FAILED ensemble_evaluator/prefect/test_prefect_ensemble.py::test_prefect_retries`

```
E           AssertionError: assert 'Failed' == 'Stopped'
E             - Stopped
E             + Failed

ert/tests/ert_tests/ensemble_evaluator/prefect/test_prefect_ensemble.py:150: AssertionError
```

Only seen for failures: `FileNotFoundError: [Errno 2] No such file or directory: 'ran_once'`. Race condition?

**To Reproduce**
1. `komodoenv` + ERT tests for `2.28.0rc0`
1.  `pytest -sxv --count=1000 ert/tests/ert_tests/ensemble_evaluator/prefect/test_prefect_ensemble.py::test_prefect_retries`

Failure rate seems like 3%

**Enviromment**
 - OS: RHEL7/jenkins both 12XX node and 11XX node
 - ERT/Komodo Release: 2021.10.rc0
 - Python version: 3.6
 - Remote/HPC execution involved: no

--- a/tests/ert_tests/ensemble_evaluator/prefect/test_prefect_ensemble.py
+++ b/tests/ert_tests/ensemble_evaluator/prefect/test_prefect_ensemble.py
@@ -125,7 +125,11 @@ def test_prefect_retries(
         if not run_path.exists():
             run_path.touch()
             raise RuntimeError("This is an expected ERROR")
-        run_path.unlink()
+        try:
+            run_path.unlink()
+        except FileNotFoundError:
+            # some other real beat us to it
+            pass
         return []
 
     pickle_func = cloudpickle.dumps(function_that_fails_once)

